src/java/org/apache/cassandra/db/ColumnFamilySerializer.java

```diff
@@ -22,6 +22,7 @@
 import java.util.UUID;
 
 import org.apache.cassandra.config.Schema;
+import org.apache.cassandra.db.filter.PageToken;
```

Comment:
nit - looks like an unused import - and you probably could revert the fomatting changes (I assume the existing code didn't conform to the format)